### PR TITLE
Start splitting of rhel7 CIS

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- create new policy for CIS
- create 3 new profiles based on levels in this policy
- change the "cis" profile to be the level2 server
- implement section 1 of CIS benchmark

#### Rationale:

update of rhel7 cis and split it to allow more granularity

### CODE REVIEW

**Review by yuumasato — CHANGES_REQUESTED** [review]
Found a few issues but it is looking good.

> * create 3 new profiles based on levels in this policy
> 
> * change the "cis" profile to be the level2 server

I like the format of the new profile IDs, and the profile id inconsistency annoys me a bit.

Idea: What if we added  the `cis_server_l2` profile,  and kept the original `cis.profile` as an extension of `cis_server_l2` profile?
The plain `cis` profile would work as a "symlink" for automation compatibility and could have "deprecated" in the title. The deprecated profile could be removed after a certain number of releases.

File: controls/cis_rhel7.yml [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+    - var_selinux_policy_name=targeted
+
+  - id: 1.6.1.4
+    title: Ensure the SELinux mode is enforcing or permissive (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    notes: >-
+      This control covers also 1.6.1.5, set the variable appropriately.
```
[the remark this answers is not in the document]
Comment (cipherboy) [reply controls/cis_rhel7.yml:440]: JFTR, this is the approach we took with AppArmor policy for the corresponding Ubuntu items (a variable with two values). HTH :-)

**Comment by yuumasato** [comment]
> I like the format of the new profile IDs, and the profile id inconsistency annoys me a bit.

So, the annoyance of changing profile IDs in already deployed system is much higher than the having inconsistency IDs.

> Idea: What if we added the `cis_server_l2` profile, and kept the original `cis.profile` as an extension of `cis_server_l2` profile?
> The plain `cis` profile would work as a "symlink" for automation compatibility and could have "deprecated" in the title. The deprecated profile could be removed after a certain number of releases.

The existence of two CIS L2 Server profiles can cause problems and confusions too. There would be duplicates showing up on SCAP Workbnench and Oscap Anaconda Addon.

Given the above, the less disruptive approach is to keep the `cis` profile as the Level 2 Server.

File: controls/cis_rhel7.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yuumasato) [reply controls/cis_rhel7.yml:440]: I see, thanks for sharing.

We discussed this yesterday and we will go a different path, which is to strongly recommend (by selecting enforcing by default) to use SELinux in enforcing mode, even on L1.
If some one really needs to use permissive mode, they will need to tailor the profile. 
@vojtapolasek Could you reflect this decision in the `notes` section?

File: products/rhel7/profiles/cis_workstation_l2.profile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+title: 'CIS Red Hat Enterprise Linux 7 Benchmark: Level 2 - Workstation'
+
+description: |-
+    This profile defines a baseline that aligns to the Center for Internet Security®
+    Red Hat Enterprise Linux 7 Benchmark™, v3.1.1, released 05-21-2021.
+
+    This profile includes Center for Internet Security®
+    Red Hat Enterprise Linux 7 CIS Benchmarks™ content. It represents the configuration profile "Level 2 - Workstation".
```
Comment (yuumasato) [inline products/rhel7/profiles/cis_workstation_l2.profile:19]: Nitpick in verbiage.
```suggestion
    Red Hat Enterprise Linux 7 CIS Benchmarks™ content.
```

File: products/rhel7/profiles/cis_workstation_l2.profile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        - yuumasato
+
+reference: https://www.cisecurity.org/cis-benchmarks/#red_hat_linux
+
+
+title: 'CIS Red Hat Enterprise Linux 7 Benchmark: Level 2 - Workstation'
+
+description: |-
+    This profile defines a baseline that aligns to the Center for Internet Security®
+    Red Hat Enterprise Linux 7 Benchmark™, v3.1.1, released 05-21-2021.
```
Comment (yuumasato) [inline products/rhel7/profiles/cis_workstation_l2.profile:16]:
```suggestion
    This profile defines a baseline that aligns to configuration "Level 2 - Workstation" from the Center for Internet Security®
    Red Hat Enterprise Linux 7 Benchmark™, v3.1.1, released 05-21-2021.
```

**Review by yuumasato — CHANGES_REQUESTED** [review]
The addition of the last sentence in the profile description seems out of place to me.
I made a suggestion to incorporate it to the first paragraph, where we state the profile alignment to the benchmark.

Other aspects look good to me.

**Comment by vojtapolasek** [comment]
Hello @yuumasato  I changed all descriptions. I also removed the second paragraph, I think it only repeated something what is mentioned in the first paragraph.

**Comment by yuumasato** [comment]
> Hello @yuumasato I changed all descriptions. I also removed the second paragraph, I think it only repeated something what is mentioned in the first paragraph.

@vojtapolasek The second paragraph was the credit attribution to CIS. I think we should keep it.

**Comment by vojtapolasek** [comment]
I hope I got it right this time :-D

**Review by yuumasato — APPROVED** [review]
Looks great to me, thank you @vojtapolasek for starting the split and updated of the CIS profiles.

**Comment by yuumasato** [comment]
@vojtapolasek Sorry, I just noticed that the Ansible Roles lint is complaining about colons `:` in the description.

```
The offending line appears to be:

  author: ComplianceAsCode development team
  description: CIS Red Hat Enterprise Linux 7 Benchmark: Level 2 - Server
                                                       ^ here  
```

**Comment by vojtapolasek** [comment]
FixedH

**Review by yuumasato — APPROVED** [review]

**Comment by alexhaydock** [comment]
Great to see that this has been merged! I'm aiming to put in some work soon to refactor my #6976 PR to do the same thing but for RHEL 8.

**Pull request merged**